xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1187,8 +1187,15 @@ int pre_header_feeding(am_private_t *para, am_packet_t *pkt)
         } else if ((CODEC_TAG_WVC1 == para->video_codec_tag)
                 || (CODEC_TAG_VC_1 == para->video_codec_tag)
                 || (CODEC_TAG_WMVA == para->video_codec_tag)) {
-            CLog::Log(LOGDEBUG, "CODEC_TAG_WVC1 == para->video_codec_tag");
-            ret = wvc1_write_header(para, pkt);
+            if (para->extrasize > 4 && !*para->extradata && !*(para->extradata + 1) &&
+                *(para->extradata + 2) == 0x01 && *(para->extradata + 3) == 0x0f && ((*(para->extradata + 4) & 0x03) == 0x03)) {
+              CLog::Log(LOGDEBUG, "CODEC_TAG_WVC1 == para->video_codec_tag, using wmv3_write_header");
+              ret = wmv3_write_header(para, pkt);
+            }
+            else {
```

Comment:
same

Reply:
I used the same indentation as is used in AMLCodec.cpp.

Reply:
@afl1 we all (including me) are walking into the trap and continuing code style of the file we're looking at. But we (team kodi) want to go the way of a unique code style step by step, means even me is trying to renew code if I touch an old place.

Nothing personal / no bikeshedding -> pls. be so nice and follow @rechi's suggestion and we're done. I know how it feels sometimes, but at the ent we're all happy :-)

Reply:
@peak3d you are a bit late, already fixed